test/functional/JIT_Test/src/jit/test/jitserver/JITServerTest.java

```diff
@@ -490,4 +594,78 @@ public void testServerComesUpAfterClientAndGoesDownAgain() throws IOException, I
 		logger.info("Stopping client...");
 		destroyAndCheckProcess(client, clientBuilder);
 	}
+
+	public void testServerSCC() throws IOException, InterruptedException {
+                logger.info("running testServerSCC: INFO and above level logging enabled");
+
+		// Run this tests only for the test variation with AOT Cache option specified
+		if(System.getProperty("CLIENT_PROGRAM").contains(aotCacheOption)) {
+			redirectProcessOutputs(clientBuilder, "testServerSCC.client");
+			redirectProcessOutputs(serverBuilder, "testServerSCC.server");
+
+			updateJITServerPort();
+
+			final Process server = startProcess(serverBuilder, "server");
+
+			Thread.sleep(SERVER_START_WAIT_TIME_MS);
+
+			Process client = startProcess(clientBuilder, "client");
+
+			logger.info("Waiting for " + CLIENT_TEST_TIME_MS + " millis.");
+			Thread.sleep(CLIENT_TEST_TIME_MS);
+
+			logger.info("Check whether the cache test_jitscc exists.");
+			if(checkCacheExists("test_jitscc"))
+				logger.info("Check if AOT methods exists in the cache test_jitscc::" + checkAOTCompileExists("test_jitscc"));
+			else
+				AssertJUnit.fail("The cache test_jitscc is not created.");
+
+			logger.info("Stopping client...");
+			destroyAndCheckProcess(client, clientBuilder);
+
+			logger.info("Waiting for " + CLIENT_TEST_TIME_MS + " millis.");
```

Comment:
We don't need to wait here after the client has been destroyed

Reply:
Currently all the existing tests which calls, `destroyAndCheckProcess` to destroy the client process waits for the specified time, possibly to ensure the resources associated with the process is cleared up properly. Hence I just followed the same code to ensure nothing breaks.

Reply:
> Currently all the existing tests which calls, destroyAndCheckProcess to destroy the client process waits for the specified time

I don't see that. Waiting is done to allow the client to compile methods.
In a couple of cases there is waiting after destroying the server process to make sure that the client can continue to compile without a server.

Reply:
That's correct I removed the unnecessary waits.